assets/airports/efhk.json

```diff
@@ -0,0 +1,1565 @@
+                ["KUDOP", "HK874", "HK873", "HK851", "HK852", "PODOM"],
+                ["KUDOP", "HK886", "ADBIX", "HK887", "HK883", "TEDMU"]
+            ]
+        },
+        "ROPAM": {
+            "icao": "ROPAM",
+            "name": "ROPAM",
+            "suffix": {"04L":"1B", "04R":"1R", "15":"1M", "22L":"2A", "22R":"2V", "33":"1W"},
+            "entryPoints": {
+                "ROPAM": [["ROPAM", "S250-"]]
+            },
+            "body": ["HK711"],
+            "rwy": {
+                "04L": ["HK711", ["HK804", "A30+|S220-"], "HK801", ["PODOM", "A23+|S210-"]],
+                "04R": ["HK711", ["HK894", "A33+|S220-"], ["HK814", "A33+|S220-"], "HK811", ["GEDLO", "A33+|S210-"]],
+                "15" : [["HK817", "A30+"], ["SUTAX", "A20+|S220-"], ["HK822", "A20+|S220-"], "HK821", ["ELKUN", "A20+|S210-"]],
+                "22L": [["HK906", "A30+|S220-"], "HK904", ["BIMVU", "A30+|S210-"]],
+                "22R": [["HK881", "A30+|S220-"], "HK883", ["TEDMU", "A20|S210-"]],
+                "33" : [["HK844", "A30+|S220-"], "HK861", ["OBONU", "A30+|S210-"]]
+            },
+            "draw": [
+                ["ROPAM"],
+                ["ROPAM", "HK711", "HK804", "HK801", "PODOM"],
+                ["HK711", "HK894", "HK814", "HK811", "GEDLO"],
+                ["ROPAM", "HK817", "SUTAX", "HK822", "HK821", "ELKUN"],
+                ["ROPAM", "HK906", "HK904", "BIMVU"],
+                ["ROPAM", "HK881", "HK883", "TEDMU"],
+                ["ROPAM", "HK844", "HK861", "OBONU"]
+            ]
+        },
+        "VEPIN": {
+            "icao": "VEPIN",
+            "name": "VEPIN",
+            "suffix": {"04L":"1B", "04R":"1R", "15":"1M", "22L":"2A", "22R":"2V", "33":"1W"},
+            "entryPoints": {
+                "VEPIN": [["VEPIN", "A100+"]]
+            },
+            "body": [["MAMOP", "A70+|S250-"]],
+            "rwy": {
+                "04L": ["HK807", "HK711", ["HK804", "A30+|S220-"], "HK801", ["PODOM", "A23+|S210-"]],
+                "04R": ["HK807", "HK711", ["HK894", "A33+|S220-"], ["HK814", "A33+|S220-"], "HK811", ["GEDLO", "A33+|S210-"]],
+                "15" : [["HK888", "A30+"], ["HK822", "A20+|S220-"], "HK821", ["ELKUN", "A20+|S210-"]],
+                "22L": [["HK892", "A30+|S220-"], "HK904", ["BIMVU", "A30+|S210-"]],
+                "22R": [["HK893", "A30+|S220-"], "HK883", ["TEDMU", "A20|S210-"]],
+                "33" : [["HK844", "A30+|S220-"], "HK861", ["OBONU", "A30+|S210-"]]
+            },
+            "draw": [
+                ["VEPIN", "MAMOP"],
+                ["MAMOP", "HK807", "HK711", "HK804", "HK801", "PODOM"],
+                ["HK711", "HK894", "HK814", "HK811", "GEDLO"],
+                ["MAMOP", "HK888", "HK822", "HK821", "ELKUN"],
+                ["MAMOP", "HK892", "HK904", "BIMVU"],
+                ["MAMOP", "HK893", "HK883", "TEDMU"],
+                ["MAMOP", "HK844", "HK861", "OBONU"]
+            ]
+        }
+    },
+
+    "spawnPatterns": [
+        {
```

Comment:
@ij1 Interesting. I would say you're right; it should reject it because it's assigned `22L`, which is invalid for that procedure. This would force the user to either use the `expect` (`e`) command to reassign the arrival runway, and then issue the new star, or (more realistically) issue the star _with_ the suffix to explicitly do both at the same time.